ably/types/options.py

```diff
@@ -208,9 +209,25 @@ def __get_rest_hosts(self):
         if fallback_hosts is None:
             if host == Defaults.rest_host or self.fallback_hosts_use_default:
                 fallback_hosts = Defaults.fallback_hosts
+            elif environment != 'production':
+                fallback_hosts = Defaults.get_environment_fallback_hosts(environment)
             else:
                 fallback_hosts = []
 
+        # Explicit warning about deprecating the option
+        if self.fallback_hosts_use_default:
+            if environment != Defaults.environment:
+                warnings.warn(
+                    "There is no longer need to set fallback_hosts_use_default,"
```

Comment:
I would suggest:

`"It is no longer required to set 'fallback_hosts_use_default', the correct fallback hosts are now inferred from the environment, 'fallback_hosts': {}"`

Reply:
https://github.com/ably/ably-python/pull/198/commits/b30059d2cd07ef62de8b28dd53c58f462e3ea3a5

Reply:
Looks good to me :-)